### ISSUE TITLE
Is `mold` need to be built from source everytime

### ISSUE THREAD

**Issue by ClSlaid** [Warchant/setup-mold#1]
Hello maintainer, I'm new to GitHub actions and looking forward to using your action script in my CI.
However, when using the action I noticed that the `mold` linker is built from source. And the log seems to mention `cache`, is there any plans to make users able to download the binary file directly? That will save a lot of time.

**Comment by Warchant** [comment]
Yes, initial plan was to cache mold binary. The [code does this](https://github.com/Warchant/setup-mold/blob/main/index.js#L57) but there's a cache miss every time for some (unknown) reason. 

Same cache [worked](https://github.com/Warchant/setup-sonar-scanner/blob/master/index.js#L47-L50) for my other action.

**Comment by Warchant** [comment]
Please try again v1 now @ClSlaid

**Comment by ClSlaid** [comment]
Now `mold` v1.1 is released with pre-built binaries. 🥳

**Comment by ClSlaid** [comment]
> Please try again v1 now @ClSlaid

But the `v1` is still the version published 20 days ago, you may need to publish first?

**Comment by Warchant** [comment]
https://github.com/actions/toolkit/issues/58
🤦
